flink-core/src/main/java/org/apache/flink/configuration/QueryableStateOptions.java

```diff
@@ -106,6 +106,12 @@
 			.defaultValue(0)
 			.withDescription("Number of query Threads for queryable state server. Uses the number of slots if set to 0.");
 
+	/** Enable Queryable state server and proxy. */
+	public static final ConfigOption<Boolean> ENABLE_QUERYABLE_STATE_PROXY_SERVER =
+		key("query.enable")
```

Comment:
how about changing the key to `query.server.xxx` as above, and the description to "Option whether ..." as `CheckpointingOptions.ASYNC_SNAPSHOTS`

Reply:
ENABLE_QUERYABLE_STATE_PROXY_SERVER is used to configure both QS proxy and server, so I think query.enable is more meaningful.
Make description more readable is a good idea, I will change it.